target/linux/brcm63xx/image/bcm63xx.mk

```diff
@@ -676,7 +738,9 @@ TARGET_DEVICES += HG553
 
 define Device/HG556a-A
   $(Device/bcm63xx)
-  DEVICE_TITLE := Huawei EchoLife HG556a rev A
+  DEVICE_VENDOR := Huawei
+  DEVICE_MODEL := EchoLife HG556a
+  DEVICE_VARIANT := rev A
```

Comment:
Here, I do not think we have a precedence on whether "rev A" or "A" is preferred.
Any guideline/example case on this? Otherwise, I would keep "rev A".

Reply:
https://openwrt.org/toh/huawei/hg556a say A, B, C. So I would remove the `rev`.

Reply:
Ah, so the "rev" is "brcm63xx slang". Yes, please remove it then.